spec/models/application_record_spec.rb

```diff
@@ -67,6 +67,21 @@
     end
   end
 
+  describe 'data_rows' do
+    let :call do
+      @record.data_rows
+    end
+    before :each do
+      @record = create :application_record,
+                       data: [['a question', 'an answer']]
```

Comment:
hahaha woops, I was looking at another test. Then I copied what was there to test mine without thinking. But cool yes I'll remove all that stuff.